src/Pact/Analyze/Remote/Server.hs

```diff
@@ -96,7 +96,7 @@ moduleNotFoundP = MP.string "<interactive>:"
                *> digitsP *> MP.char ':'
                *> digitsP *> MP.char ':'
                *> MP.string " Module "
-               *> fmap fromString (MP.some $ MP.notChar ' ')
+               *> fmap fromString (MP.some $ MP.anySingleBut ' ')
```

Comment:
The documentation for `anySingleBut` mentions:

> It's a good idea to attach a label to this parser.

Thoughts?

Reply:
I don't really understand this parser so I was just following guidelines for how to deal with the breaking change in MP7 removing `notChar` -- so perhaps this is better addressed by @tomsmalley or @eskimor

Reply:
`anySingleBut` is identical to `notChar` (from looking at the implementation). I don't think we need to bother with a nicer error message, we discard it at the use site anyway. LGTM :+1: